server/src/main/scala/com/cloudera/livy/utils/Json.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to Cloudera, Inc. under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Cloudera, Inc. licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cloudera.livy.utils
+
+import com.ning.http.client.Response
+import org.json4s._
+import org.json4s.jackson.JsonMethods._
+
+
+object Json extends (Response => JValue) {
+  def apply(r: Response): JValue =
+    (((r: Response) => r.getResponseBody) andThen (s => parse(StringInput(s), true)))(r)
```

Comment:
This is a little hard to read: isn't it the same as:

    parse(StringInput(r.getResponseBody(), true))

Or some other simpler code like that? Also, a whole object just for this one method seems a little overkill; it seems it's just used in one class so it could be a private method? (I might be wrong though.)